ova/Libraries/provision-opendistro.sh

```diff
@@ -0,0 +1,306 @@
+
+    eval "$sys_type install filebeat-"${ELK_VERSION}" -y -q  $debug"
+    if [  "$?" != 0  ]
+    then
+        echo "Error: Filebeat installation failed"
+        exit 1;
+    else
+        eval "curl -so /etc/filebeat/filebeat.yml https://raw.githubusercontent.com/wazuh/wazuh/${BRANCH}/extensions/filebeat/7.x/filebeat_all_in_one.yml --max-time 300  $debug"
+        eval "curl -so /etc/filebeat/wazuh-template.json https://raw.githubusercontent.com/wazuh/wazuh/${BRANCH}/extensions/elasticsearch/7.x/wazuh-template.json --max-time 300 $debug"
+        eval "chmod go+r /etc/filebeat/wazuh-template.json $debug"
+        eval "curl -s https://packages.wazuh.com/3.x/filebeat/wazuh-filebeat-0.1.tar.gz --max-time 300 | tar -xvz -C /usr/share/filebeat/module $debug"
+        eval "mkdir /etc/filebeat/certs $debug"
+        eval "cp /etc/elasticsearch/certs/root-ca.pem /etc/filebeat/certs/ $debug"
+        ls mv /etc/elasticsearch/certs/
+        eval "mv /etc/elasticsearch/certs/filebeat* /etc/filebeat/certs/ $debug"
+        # Start Filebeat
+        startService "filebeat"
+
+        logger "Done"
+    fi
+}
+
+## Kibana
+installKibana() {
+  #  eval "curl https://d3g5vo6xdbdb9a.cloudfront.net/yum/opendistroforelasticsearch-artifacts.repo -o /etc/yum.repos.d/opendistroforelasticsearch-artifacts.repo"
+
+    logger "Installing Open Distro for Kibana..."
+    eval "$sys_type install opendistroforelasticsearch-kibana-${OPENDISTRO_VERSION} -y -q $debug"
+    if [  "$?" != 0  ]
+    then
+        echo "Error: Kibana installation failed"
+        exit 1;
+    else
+        eval "curl -so /etc/kibana/kibana.yml https://raw.githubusercontent.com/wazuh/wazuh/${BRANCH}/extensions/kibana/7.x/kibana_all_in_one.yml --max-time 300 $debug"
+        eval "cd /usr/share/kibana $debug"
+
+
+
+        if [ "${STATUS_PACKAGES}" = "prod" ]; then
+          eval "sudo -u kibana /usr/share/kibana/bin/kibana-plugin install https://packages-dev.wazuh.com/Backups/staging/app/kibana/wazuhapp-${WAZUH_VERSION}_${ELK_VERSION}.zip $debug"
+        elif [ "${STATUS_PACKAGES}" = "dev" ]; then
+          eval "sudo -u kibana /usr/share/kibana/bin/kibana-plugin install https://packages-dev.wazuh.com/pre-release/ui/kibana/wazuhapp-${WAZUH_VERSION}_${ELK_VERSION}.zip $debug"
+        fi
+
+
+        eval "sudo -u kibana /usr/share/kibana/bin/kibana-plugin install https://packages-dev.wazuh.com/trash/app/kibana/wazuhapp-4.0.0_7.8.0.zip $debug"
+        if [  "$?" != 0  ]
+        then
+            echo "Error: Wazuh Kibana plugin could not be installed."
+            exit 1;
+        fi
+        eval "mkdir /etc/kibana/certs $debug"
+        eval "mv /etc/elasticsearch/certs/kibana* /etc/kibana/certs/ $debug"
+        eval "setcap 'cap_net_bind_service=+ep' /usr/share/kibana/node/bin/node $debug"
+
+        # Start Kibana
+        startService "kibana"
+
+        logger "Done"
+    fi
+}
+
+## Health check
+healthCheck() {
+
+    cores=$(cat /proc/cpuinfo | grep processor | wc -l)
```

Comment:
```suggestion
    cores=$(nproc)
```

Reply:
Done in 528feec208a367ba616fb87ea4237096d0681494